csrankings-j.csv

```diff
@@ -781,8 +781,9 @@ Jens-Peter Redlich,Humboldt University of Berlin,http://sar.informatik.hu-berlin
 Jeong hwan Jeon,UNIST,https://rml.unist.ac.kr,ZTe13LgAAAAJ
 Jeong-Gil Ko,Yonsei University,http://www.eis-lab.org,5zP95ZQAAAAJ
 Jeong-Hyon Hwang,University at Albany - SUNY,https://www.albany.edu/ceas/jeong-hyon-hwang.php,uurRDOwAAAAJ
+Jeonggeun Kim,Kyungpook National University,https://cse.knu.ac.kr/,lvU4X6IAAAAJ

```

Comment:
Not an actual home page for this faculty member, please fix.

Reply:
Updated the home page.